quartz/styles/base.scss

```diff
@@ -76,6 +77,16 @@ a {
       padding: 0;
     }
   }
+
+  &.external .external-icon {
+    text-align: bottom;
+    height: 1ex;
+    margin-left: 0.15em;
```

Comment:
let's also add a small right margin as well

Reply:
I personally don't feel like that's necessary. There's already a spacing, and I feel like adding more makes it look weird; the spacing on each side is inconsistent.

Reply:
can you inspect the padding? this reads as if the spacing is inconsistent if you only add margin on the left

Reply:
Well, without the margin, it looks weird because the icon is squished against the text. One on the right isn't needed, because we probably want to retain the same spacing on both sides of the `<a>`.

Reply:
imo it feels squished without the margin, please add it in, i'd like to maintain creative control over the design of the default style and i'll adjust if needed

Reply:
if you say so...

here's what it looks like without right margin:
![image](https://github.com/jackyzha0/quartz/assets/146651411/623cb735-96cc-4c1b-beab-3cd99f89c373)

and with:
![image](https://github.com/jackyzha0/quartz/assets/146651411/a1bba6f9-bf6d-476d-8a8f-2826af9644b3)